staging/src/k8s.io/dynamic-resource-allocation/deviceclass/extendedresourcecache/extendedresourcecache.go

```diff
@@ -162,6 +177,23 @@ func (c *ExtendedResourceCache) updateMapping(newDeviceClass, oldDeviceClass *re
 		"deviceClass", newDeviceClass.Name)
 }
 
+// updateClassMapping updates the cache with the device class mapping.
+func (c *ExtendedResourceCache) updateClassMapping(deviceClass *resourceapi.DeviceClass) {
+	if deviceClass == nil {
+		return
+	}
```

Comment:
Why the nil check?

As far as I can tell it's always called with a valid instance.

Might be worth removing also for the other methods, I missed that earlier.

Reply:
removed the nil check in this and other methods